src/utils/locale.spec.ts

```diff
@@ -35,7 +36,7 @@ describe("NumberStringFormat", () => {
   });
 
   locales.forEach((locale) => {
-    it(`integers localize and delocalize in "${locale}"`, () => {
+    it(`locale: integers localize and delocalize in "${locale}"`, () => {
```

Comment:
You could place these in a `describe("locale", () => { /*...*/` block for grouping instead of updating the individual test names.